### ISSUE TITLE
Repeat behavior restarting the animation.

### ISSUE THREAD

**Issue by JacobSkyrme** [XamlAnimatedGif/XamlAnimatedGif#111]
Is there a way to set repeat behavior at the end of the animation? Currently when ever i change the repeat behavior it restarts the animation and displays it really weirdly: https://gyazo.com/892f7de9a20cfe3e9b544727deb7461e 

The bug only occurs when setting repeat behavior to a value. I've tried having the button on the same WPF but it makes no difference. I've tried the code within the button and in methods but makes no difference either.

Basically I'm trying to figure a way to change animation sources at the end of an animation which is repeating infinantly.

Thanks for your help - Jacob

**Comment by thomaslevesque** [comment]
Hi @JacobSkyrme,

You could set the repeat behavior to 1x, and use the `Animator.AnimationCompleted` event to detect when the animation completes. To get the Animator, use `AnimationBehavior.GetAnimation(imgControl)`.

**Comment by thomaslevesque** [comment]
Closing due to inactivity
